test/test_language_scanner.py

```diff
@@ -203,7 +203,7 @@ def test_javascript_package_none_found(self, filename: str) -> None:
     @pytest.mark.parametrize(
         "filename",
         [
-            (str(TEST_FILE_PATH / "FAIL-PKG-INFO")),
+            # (str(TEST_FILE_PATH / "FAIL-PKG-INFO")),
```

Comment:
Yeah, it's not clear to me that this test is useful any more either.  Rather than commenting it out, let's delete this line *and* delete the test file if it won't be in use any longer.